lib/travis/github_apps.rb

```diff
@@ -43,6 +44,7 @@ def initialize(installation_id, config = {})
 
       @accept_header       = config.fetch(:accept_header, "application/vnd.github.machine-man-preview+json")
       @installation_id     = installation_id
+      @repository_id       = repository_id
```

Comment:
It is not clear to me by reading the code what kind of data `repository_id` is supposed to be passed when initializing this object.